src/Tizen.Applications.Common/Tizen.Applications.RPCPort/Port.cs

```diff
@@ -96,5 +96,17 @@ public void UnshareFile()
             if (err != Interop.LibRPCPort.ErrorCode.None)
                 throw new InvalidIOException();
         }
+
+        /// <summary>
+        /// Disconnects the port.
+        /// </summary>
+        /// <exception cref="InvalidIOException">Thrown when an internal IO error occurrs.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        public void Disconnect()
+        {
+            Interop.LibRPCPort.ErrorCode err = Interop.LibRPCPort.Port.Disconnect(Handle);
```

Comment:
The Interop.LibRPCPort.Port.Disconnect() returns the RPC_PORT_ERROR_INVALID_PARAMETER or the RPC_PORT_ERROR_NONE.
I think the exception should be changed to "ArgumentNullException".

Reply:
ArgumentNullException should throw when argument is existed and it is wrong.
but this method there is no argment

Reply:
I got it. Thanks.

Reply:
I fixed it.